packages/flutter_test/lib/src/binding.dart

```diff
@@ -106,6 +111,32 @@ mixin TestDefaultBinaryMessengerBinding on BindingBase, ServicesBinding {
   }
 }
 
+/// An accessibility announcement made by the Flutter framework.
+///
+/// This class is mostly used by the testing api to store the announcements
```

Comment:
nit: s/mostly/intended to be/
nit: s/api/API/

Reply:
Done!